ocf_datapipes/select/trim_dates_with_insufficent_data.py

```diff
@@ -0,0 +1,56 @@
+import logging
+from datetime import datetime
+
+import numpy as np
+import xarray as xr
+from torchdata.datapipes import functional_datapipe
+from torchdata.datapipes.iter import IterDataPipe
+
+logger = logging.getLogger(__name__)
+
+
+@functional_datapipe("drop_systems_with_lessthan_oneday_data")
+class TrimDatesWithInsufficentDataIterDataPipe(IterDataPipe):
+    """Trim the date values of the Xarray Timeseries data"""
+
+    def __init__(self, source_datapipe: IterDataPipe, intervals: int):
```

Comment:
Maybe we can set the interval to default to 288, for the 5min data as that is our default.
```suggestion
    def __init__(self, source_datapipe: IterDataPipe, intervals: int = 288):
```